packages/keyboard-e2e/test/assertions/isActiveElement.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable valid-typeof */
```

Comment:
I won't port this across to `vets-website`, but needed a way to do this type of assertion for the proof-of-concept.